File: apollo/src/main/java/io/muun/apollo/domain/errors/IncorrectRecoveryCodeError.java
package io.muun.apollo.domain.errors;

import io.muun.apollo.external.UserFacingErrorMessages;

public class IncorrectRecoveryCodeError extends UserFacingError {

    public IncorrectRecoveryCodeError() {
        super(UserFacingErrorMessages.INSTANCE.incorrectRecoveryCode());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/IncorrectRecoveryCodeError.kt
package io.muun.apollo.domain.errors

import io.muun.apollo.data.external.UserFacingErrorMessages

class IncorrectRecoveryCodeError :
    UserFacingError(UserFacingErrorMessages.INSTANCE.incorrectRecoveryCode()) {
    override val classification = ErrorClassification.EXPECTED
}
